.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Integration Test
+
+on:
+  push:
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build emissions-api
+        run: docker-compose -f .test/docker-compose.yml build emissionsapi
+      - name: Spin up database and emissions-api web server
+        run: docker-compose -f .test/docker-compose.yml up -d
+      - name: Wait for database
+        run: until PGPASSWORD=user psql -h 127.0.0.1 -U user db -c '\q'; do sleep 1; done
+      - name: Check if emissions-api web server is up
+        run: nc -z 127.0.0.1 8000
+      - name: Download and include data to the emissions-api service
+        run: docker-compose -f .test/docker-compose.yml exec -T emissionsapi /opt/emissions-api/venv/bin/emissionsapi-autoupdater
+      - name: Products API
+        run: 'curl --fail -X GET "http://localhost:8000/api/v2/products.json" -H  "accept: application/json"'
```

Comment:
Would it make sense to verify that the result is valid JSON?
E.g. something like `... | jq`?

Reply:
I added a script with a little bit more sophisticated testing, inclusive `jq`.